std/traits.d

```diff
@@ -2684,7 +2686,7 @@ private template hasRawAliasing(T...)
     enum hasRawAliasing = Impl!(RepresentationTypeTuple!T);
 }
 
-///
+//
```

Comment:
Why are we undocumenting useful example blocks?

Reply:
Because `hasRawAliasing` is a private symbol -> https://github.com/dlang/phobos/blob/v2.076.0/std/traits.d#L2661

Reply:
Oh, OK. So this was an "example" for a private symbol.